src/discordcr/mappings/gateway.cr

```diff
@@ -104,7 +104,7 @@ module Discord
         status: {type: String, nilable: true, emit_null: true},
         game: {type: GamePlaying, nilable: true, emit_null: true},
         afk: Bool,
-        since: Int64
+        since: {type: Int64, nilable: true, emit_null: true}
```

Comment:
We should really replace all the `Int64, nilable: true` with `Int64?` but that should be a whole new PR.